packages/components/src/flyout/flyout-content/hook.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { css, cx } from '@emotion/css';
+
+/**
+ * Internal dependencies
+ */
+import { useContextSystem } from '../../ui/context';
+import * as styles from '../styles';
+
+/**
+ * @param {import('../../ui/context').PolymorphicComponentProps<import('../types').ContentProps, 'div'>} props
+ */
+export function useFlyoutContent( props ) {
+	const {
+		className,
+		elevation = 5,
+		maxWidth = 360,
+		...otherProps
+	} = useContextSystem( props, 'FlyoutContent' );
+
+	const classes = cx( styles.FlyoutContent, css( { maxWidth } ), className );
+
+	return {
+		...otherProps,
+		elevation,
+		classes,
+	};
+}
```

Comment:
We'll need to refactor FlyoutContent to use `@emotion/styled` and `@emotion/react`.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/32197/commits/3624b7794b3b095d05619ee4053e86b81abc1d53 and https://github.com/WordPress/gutenberg/pull/32197/commits/3598b735319eaedc5b498e7d470a3b9f2d527243